src/main/kotlin/org/jetbrains/research/testspark/helpers/psiHelpers/PsiHelperFactory.kt

```diff
@@ -1,14 +1,19 @@
 package org.jetbrains.research.testspark.helpers.psiHelpers
 
+import com.intellij.openapi.diagnostic.Logger
 import com.intellij.psi.PsiFile
 import com.intellij.psi.PsiJavaFile
 
 object PsiHelperFactory {
-    fun getPsiHelper(psiFile: PsiFile): PsiHelperInterface {
+    fun getPsiHelper(psiFile: PsiFile?): PsiHelper {
+        val log = Logger.getInstance(this::class.java)
         return when (psiFile) {
-            is PsiJavaFile -> JavaPsiHelper()
+            is PsiJavaFile -> {
+                log.info("Getting psi helper for JAVA")
+                JavaPsiHelper(psiFile)
+            }
             // TODO implement KotlinPsiHelper class
-//            is KtFile -> KotlinPsiHelper()
+            // is KtFile -> KotlinPsiHelper(psiFile)
             else -> throw IllegalArgumentException("Unsupported file type")
```

Comment:
The plugin must not throw any exceptions to the users. Either no reaction at all, or a pop-up window with a warning/error message.

Reply:
No reaction is not a good idea, but a popup window is good. Do we have an upper-level try-catch block that converts exceptions into balloon popup error notifications?

Reply:
That would have been nice to have this pipeline of actions. Exception --> Pop-up window. If this does not exist, then it should be a different PR. Now I will just do nothing, as it actually was before